### DISCUSSION TITLE
Settings from the transient analysis component don't seem to work. How can I make sure the correct options will be selected by spice?

### DISCUSSION THREAD

**Discussion by thinkerer101** [ra3xdh/qucs_s#612] [Q&A] [answered]
I'm using the current stable release of Qucs-S, 24.1, and ngspice current release, 42. ngspice-42 isn't in the system path because my distro ships an older version. But I have already verified this issue with other versions, i.e., when I select an option like TEMP in the settings of transient analysis, it is not changed in the simulation. The only way to get TEMP, METHOD, etc. right is with the OPTIONS section component. Is this the desired behavior?

**Comment by ra3xdh** [comment, marked as answer]
It is not a bug. Ngspice doesn't support temperature specification in the transient simulation (or any other simulation) parameters. You always have to use the global option to set the schematic temperature.

**Comment by thinkerer101** [comment]
Thanks for your answer. If I may ask, could these global parameter settings get a menu entry option under Simulation? It's what someone new to Qucs-S but coming from PSpice/LTSpice would look for.
